packages/beacon-node/src/network/network.ts

```diff
@@ -350,4 +356,20 @@ export class Network implements INetwork {
       }
     }
   };
+
+  private async onLightclientFinalityUpdate(finalityUpdate: altair.LightClientFinalityUpdate): Promise<void> {
+    try {
+      return await this.gossip.publishLightClientFinalityUpdate(finalityUpdate);
+    } catch (e) {
+      this.logger.debug("Error on BeaconGossipHandler.onLightclientFinalityUpdate", {}, e as Error);
+    }
+  }
+
+  private async onLightclientOptimisticUpdate(optimisticUpdate: altair.LightClientOptimisticUpdate): Promise<void> {
+    try {
+      return await this.gossip.publishLightClientOptimisticUpdate(optimisticUpdate);
+    } catch (e) {
+      this.logger.debug("Error on BeaconGossipHandler.onLightclientOptimisticUpdate", {}, e as Error);
```

Comment:
consciously have this at debug level to prevent unnecessary flooding of logs with `PublishError.InsufficientPeers` which will be the case given there are likely no subscribers to this topics yet

Reply:
does a full node always publish an optimistic update message? I see this in the spec

```
Whenever fork choice selects a new head block with a sync aggregate participation >= MIN_SYNC_COMMITTEE_PARTICIPANTS and a post-Altair parent block, full nodes with at least one validator assigned to the current sync committee at the block's slot SHOULD broadcast derived light client data as follows
```

Reply:
also it's important not to publish these messages before 1/3 of slot according to the spec
```
These messages SHOULD be broadcasted after one-third of slot has transpired
```

Reply:
i agree, this check should be done before emitting the event at `onSyncAggregate` function, optimistic update is the powerful indicator of where chain might finalize rather than just where the head is,